imports/ui/components/documents/show/document-show.html

```diff
@@ -14,8 +14,20 @@
           {{#if problem.claimed}}
           <span class="badge badge-success">CLAIMED</span>
           {{/if}}
-          <h2>{{problem.summary}}</h2>
-          <p>{{problem.description}}</p>
+
+
+          {{#if isProblemOwner problem.createdBy}}
```

Comment:
@samuelralak These wont be required, I have already added the buttons. 
Please check 
https://github.com/EmurgoHK/Emurgis/blob/master/imports/ui/components/documents/show/document-show.html#L9

Reply:
I had done that as part of [#63](https://github.com/EmurgoHK/Emurgis/issues/63). not to worry though, my next PR should fix all that will have been deleted